### ISSUE TITLE
Improve the accessibility of the dropdown / combobox components

### ISSUE THREAD

**Issue by ndorfin** [MathewBoyles/locations-of-interest#1]
Hi Matt and Chandler,

The `react-select` component has a rather poor history of [failing several accessibility concerns](https://github.com/JedWatson/react-select/issues?q=is%3Aissue+is%3Aopen+accessibility)

I'd recommend moving to a good ol'`<select>` element, and/or using checkboxes for selecting many regions.

Cheers,
-S
